rero/rero-ils
# translations: adds editor translations support

* Removes JSONSchema backend on the fly translations.
* Removes legacy `validationMessage` JSONSchema` property.
* Adds suffix to the validation messages keys.
* Fixes missing punctuation in the validation messages.
* Cleans babel configuration.
* Removes legacy schema form code.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- FixesL https://tree.taiga.io/project/rero21-reroils/task/1456?kanban-status=1224894

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* https://github.com/rero/ng-core/pull/187
* https://github.com/rero/rero-ils-ui/pull/277

## How to test?

- Go to the professionnal interface
  - the language menu should have the same behaviour than the public interface
  - test the editor with several resources, switch the language, the label, the description the placeholder and the validation should be translated.

## Code review check list
